pyproject.toml

```diff
@@ -13,25 +13,25 @@ name = "frequenz-api-microgrid"
 description = "Frequenz gRPC API for monitoring and control of microgrids"
 readme = "README.md"
 license = { text = "MIT" }
-keywords = [ "frequenz", "api", "microgrid", "grpc" ]
+keywords = ["frequenz", "api", "microgrid", "grpc"]
 classifiers = [
-   "Development Status :: 3 - Alpha",
-   "Intended Audience :: Developers",
-   "License :: OSI Approved :: MIT License",
-   "Programming Language :: Python :: 3",
-   "Programming Language :: Python :: 3 :: Only",
-   "Topic :: Software Development :: Libraries",
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Topic :: Software Development :: Libraries",
```

Comment:
There are a few formatting changes in the same commit as the primary one. Any chance we can place the formatting changes in a different commit?

Reply:
Ah, yeah, I didn't realize because I have an auto-formatting when saving, sorry about that. Will do!

Reply:
Updated!

At some point I want to plug this to the CI so we ensure consistent formatting of toml files too :)

https://github.com/frequenz-floss/frequenz-repo-config-python/issues/15

Reply:
Nice, thanks!

> At some point I want to plug this to the CI so we ensure consistent formatting of toml files too :)

Makes sense.